vendor/bat-native-ads/src/bat/ads/internal/frequency_capping/exclusion_rules/split_test_frequency_cap.cc

```diff
@@ -57,8 +57,8 @@ bool SplitTestFrequencyCap::DoesRespectCap(const CreativeAdInfo& ad) const {
   }
 
   if (ad.split_test_group.empty()) {
-    // Always respect cap if there is no split testing group in the catalog
-    return true;
+    // Do not respect cap if there is no split testing group in the catalog
```

Comment:
we should flip the return value for the conditional in L56

Reply:
Discussed issue on DM, pushed changes